docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md

```diff
@@ -0,0 +1,79 @@
+# Use GitHub as main hub for the Opinionated Digital Center
+
+* Status: proposed
+* Date: 2020-04-02
+
+## Context and Problem Statement
+
+We want to share the Opinionated Digital Center's (ODC) opinions, code, templates,
+pipelines and tools with the community. We want to make it as easy as possible to access
+and to contribute to, either by exchanging ideas, submitting issues and Pull/Merge
+Requests. Hence, we need to choose a hub.
+
+## Decision Drivers
+
+* Our main audience target are environments and companies using self-hosted or
+  cloud versions of GitLab or GitHub for hosting the code, and GitLab CI for pipelines.
+  To our knowledge, GitHub Actions is not yet widely available/used in the
+  companies we worked with, especially since it requires a product update,
+  which is often infrequent in these environments.
+* Having said that, GitHub Actions is fully available on the cloud and will no doubt
+  arrive in companies sooner or later.
+* Popularity and acceptance of the hub solution is key, so as not to deter
+  participation.
+
+*Note:* Whatever the option used, our templates will aim at making pipeline code and
+setup code available for both GitLab CI and GitHub Actions. The projects holding the
+templates and all other ODC projects will be using the chosen option's pipeline.
+
+## Considered Options
+
+* Main hub: GitLab; pipeline: GitLab CI; mirroring: GitHub, pull only.
+* Main hub: GitHub; pipeline: GitLab CI; mirroring: GitLab, pull only.
+* Main hub: GitHub; pipeline: GitLab CI; mirroring: GitLab, bi-directional.
+* Main hub: GitHub; pipeline: GitHub Actions; mirroring: GitLab, pull only.
+
+## Decision Outcome
+
+Chosen option: "Main hub: GitHub; pipeline: GitHub Actions; mirroring: GitLab, pull
+only", because the team could quickly enough get up to speed with GitHub Actions and
+it is sufficiently mature and reliable to use.
+
+## Pros and Cons of the Options
+
+### Main hub: GitLab; pipeline: GitLab CI; mirroring: GitHub, pull only
+
+* Good, because:
+  * Everything is in one place and fully integrated.
+* Bad, because:
+  * GitLab's popularity amongst the community is not as high as GitHub's.
+    Even our contributors, who are using GitLab daily (and liking it) at their workplace
+    turn first to GitHub for finding code and, most importantly, interacting around it.
```

Comment:
I would add that this opinion is mostly amongst open source community.
The first name that comes to mind while looking for OS projects is GitHub

Reply:
done